### PR TITLE
[17.multilingual-bot] Applying feedback to fix targeting only the first non target

### PR DESCRIPTION
**Pull request by VictorGrycuk**
Fixes #

## Specific Changes

  - 
  -
  -

### CODE REVIEW

File: samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java [hunk 1]
```diff
@@ -128,18 +129,16 @@ public MultiLingualBot(UserState withUserState) {
     }
 
     private static CompletableFuture<Void> sendWelcomeMessage(TurnContext turnContext) {
-        for (ChannelAccount member: turnContext.getActivity().getMembersAdded()) {
-            // Greet anyone that was not the target (recipient) of this message.
-            // To learn more about Adaptive Cards, see https://aka.ms/msbot-adaptivecards for more details.
-            if (member.getId() != turnContext.getActivity().getRecipient().getId()) {
+        return turnContext.getActivity().getMembersAdded().stream()
+            .filter(member -> !StringUtils.equals(member.getId(), turnContext.getActivity().getRecipient().getId()))
```
Comment (VictorGrycuk) [inline samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java:133]: We can create a variable to replace `turnContext.getActivity().getRecipient().getId()` on this line and make it more readable

**Review by fran893 — APPROVED** [review]
LGTM

File: samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java (hunk 1 above)
Comment (Batta32) [reply samples/17.multilingual-bot/src/main/java/com/microsoft/bot/sample/multilingual/MultiLingualBot.java:133]: Agree with you, but we are not re-using the `turnContext.getActivity().getRecipient().getId()` again apart from that line. In other case, I'd like you approach, but not here

**Review by Batta32 — APPROVED** [review]
LGTM. We need to validate this before merging. But it's ok

**Pull request merged**